.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/rust-blockchain/ethereum.git
 [submodule "vendor/frontier"]
 	path = vendor/frontier
-	url = https://github.com/PureStake/frontier
+	url = https://github.com/paritytech/frontier.git
```

Comment:
@tgmichel is it going to be compatible with our parachain?
if not @TheQuantumPhysicist can you see if you can use our parachain branch of frontier.

Reply:
We need to use https://github.com/purestake/frontier/tree/moonbeam

Reply:
Will do.

Reply:
This is done.